core/src/main/scala/dogs/Sorted.scala

```diff
@@ -0,0 +1,51 @@
+/**
+ * Created by anicolaspp on 4/10/16.
+ */
+
+package dogs
+
+import Predef._
+import simulacrum.typeclass
+import scala.{inline,Iterable}
+import java.lang.{String,StringBuilder}
+import scala.annotation.{tailrec}
+import dogs.syntax.birds._
+import cats._
+
+
+@typeclass trait Sorted[A] {
+  def sorted(xs: List[A]): List[A]
```

Comment:
is this the right way to define it? 

@typeclass trait Sorted[M[_]] {
  def sorted[A](xs: M[A]): List[A]
}